### PR TITLE
Stop using . in artifactId, use - instead, per the usual conventions.

### PR DESCRIPTION
**Pull request by RomainMuller**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license.

### CODE REVIEW

**Comment by kiiadi** [comment]
I still think we should drop the `aws-` prefix from the artifactId, it seems redundant. The AWS SDK for Java v2 just uses the basic service name, so for example the dependency is `software.amazon.awssdk:s3:$VERSION`. I understand there may be cases where a conflict could arise - but we should tackle the differentiation at that point rather than make every service suffer from the additional redundancy.

**Comment by RomainMuller** [comment]
Discussed off-line with @kiiadi and came to the following conclusion:
* Drop the `aws-` prefix from the `artifactId` will make the maven coordinates simpler & align with the Java SDK 2.0 (maybe allow it as a tie-breaker, but we don't actually have a conflict yet, so probably not)
* Package names for L1/L2 libraries should be `software.amazon.awscdk.services.<package name minus '@aws-cdk/aws-' prefix>`
* It might be a good idea to rename `@aws-cdk/rtv` to `@aws-cdk/runtime-values`, because the acronym is not standard & could confuse.

**Comment by RomainMuller** [comment]
Updated PR to change `artifactId`s and package names as described above, and added a `cdk-` prefix to CDK core packages.

File: examples/cdk-examples-java/pom.xml.t.js [hunk 1]
```diff
@@ -77,14 +77,35 @@ process.stdout.write(`
 
     <dependencies>
         <dependency>
-            <groupId>com.amazonaws</groupId>
+            <groupId>software.amazon.jsii</groupId>
```
Comment (kiiadi) [inline examples/cdk-examples-java/pom.xml.t.js]: out of interest - how are we releasing jsii? I noticed from the repo there that it's org.jsii - did we want to keep that? or is this an amazon-branded product?

Comment (RomainMuller) [reply examples/cdk-examples-java/pom.xml.t.js]: It'll be published under `software.amazon.jsii`, and supposedly all places were updated accordingly (somehow this one seems to have slept though).

**Pull request merged**